binary/capabilities.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package io.substrait;
+
+option java_multiple_files = true;
+option csharp_namespace = "Substrait.Protobuf";
+
+// Defines a set of Capabilities that a system (producer or consumer) supports.
+message Capabilities {
```

Comment:
Can you split this into its own PR?

Reply:
I have split it out but won't post until the other patches get merged since it depends on them (the docs part)